commons-schemavalidation/src/main/java/eu/europeana/api/commons/JsonSchemaLocation.java

```diff
@@ -0,0 +1,6 @@
+package eu.europeana.api.commons;
+
+public interface JsonSchemaLocation {
```

Comment:
This class is not needed here. 

**JsonSchemaLocation** interface only contains the classpath information which is used in the calling API and not here. JSON schema location will be specific for each API and will be present in the calling API not here.